### PR TITLE
feat(python): add Python binding for resources

### PR DESCRIPTION
**Pull request by TatianaJin**
## What's changed and what's your intention?
Add python binding for `ResourceOp` implementations.

## Checklist

- [x] I have written necessary docs and comments.
- [x] I have successfully built and tested it.

## Refer to a related PR or issue link (optional)
#152

### CODE REVIEW

**Comment by TatianaJin** [comment]
cc @MaKaili  @yang-han

**Review by yang-han — DISMISSED** [review]

File: examples/quickstart/main.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+from os import path as osp
+
+from ofnil import *
+
+home = osp.dirname(osp.abspath(__file__))
+
+
+def register_source_graph(client: Client, graph_name: str):
+    # vertices
+    reviewer = vertex_entity(f"{graph_name}_reviewer", "Reviewer", "reviewerID")
```
Comment (MaKaili) [inline examples/quickstart/main.py:10]: Import the resource information from one config file, like .yml, as a FYP task?

**Review by MaKaili — COMMENTED** [review]
LGTM.

File: examples/quickstart/main.py (hunk 1 above)
Comment (TatianaJin) [reply examples/quickstart/main.py:10]: I think we can keep the feature-as-codes concept here? @yang-han what do you think? We also have the yet-to-support functionality to retrieve source graph definitions from graph databases.

Comment (yang-han) [reply examples/quickstart/main.py:10]: config file is also fine, i think it won't break the feature-as-code, as long as the config is also managed by a version-control-system. 

But the config file itself will be tricky as there is not a standard that user can directly export the config that ofnil can parse and use, which means users will also need to learn how to write that config.

We can keep this in mind, for now, let's just set its priority to be low. (most current feature stores also don't have such config, right?)

**Review by yang-han — APPROVED** [review]

File: examples/quickstart/main.py (hunk 1 above)
Comment (MaKaili) [reply examples/quickstart/main.py:10]: I checked Feast's example, they also register the resource in the scripts. Let's move on.

**Pull request merged**